CHANGELOG.md

```diff
@@ -1,6 +1,67 @@
 # Change Log
 
-Notable changes to QMCPACK will be documented in this file. 
+Notable changes to QMCPACK will be documented in this file.
+
+## [Unreleased]
+
+### Notes
+
+### QMCPACK updates
+
+* Support for finite different linear response (FDLR) method and
+  wavefunctions, developed and contributed by Nick Blunt and Eric
+  Neuscamman, see Journal of Chemical Physics 147, 194101 (2017),
+  https://doi.org/10.1063/1.4998197 and
+  https://arxiv.org/abs/1707.09439 .
+* Major update to convert4qmc, conversion from GAMESS and other
+  gaussian basis set codes. HDF5 output is now supported for large
+  wavefunctions with -hdf5 option. Significantly improved example
+  inputs \*.qmc.in.xml.
+* Gaussian based trial wavefunctions now supported by structure of
+  arrays implementation (ENABLE_SOA=1). A full reimplementation that
+  will also support gaussians in periodic boundary conditions, e.g.
+  from pyscf, is in progress.
+* Initialization of multideterminant wavefunctions improved for faster
+  startup and lower memory usage. In practice this significantly
+  raises the usable maximum number of determinants.
+* Maximum CPU time setting (maxcpusecs): QMC drivers will not
+  start a new block if there is not enough estimated time remaining to
+  complete the next block and gracefully shut down.
+* Homogeneous electron gas wavefunction support and tests.
+* New command line verbosity command line flag -verbosity. Output of
+  QMCPACK will be overhauled over the next few releases to
+  support low, high, and debug options, and also to significantly improve
+  readability and utility.
+* Bugfix: Umrigar drift diffusion term is now consistent with the
+  Umrigar small time step error algorithm with complex wavefunctions.
+* Bugfix: Momentum distribution is now correctly weighted and also
+  correctly signed for twist averaging.
+* Bugfix: DMC with reconfiguration corrected
```

Comment:
There is still issue with reconfiguration. The results are not correct as the dynamic population runs.

Reply:
I'll remove this entry. Does Andrew know what the problem is?

Reply:
Yes. I have told him the issue and hand him my numbers and input blocks.